tests/conftest.py

```diff
@@ -69,6 +70,10 @@ def named_servers(app):
 def dockerspawner_configured_app(app, named_servers):
     """Configure JupyterHub to use DockerSpawner"""
     app.config.DockerSpawner.prefix = "dockerspawner-test"
+    # If not a standard release e.g. (2, 0, 0, 'rc4', '') use full tag
+    if len(jh_version_info) > 3 and jh_version_info[3]:
```

Comment:
I think this can be simpler to use just `jupyterhub.__version__`, which will always be the right string.